src/pages/c/[configKind]/[tip].astro

```diff
@@ -7,7 +7,11 @@ import CodeWindow from "../../../components/CodeWindow/CodeWindow.astro";
 import SocialBar from "../../../components/UI/SocialBar.astro";
 import Link from "../../../components/Link.astro";
 import Icon from "astro-icon";
-import BreadCrumb, { BreadCrumbList } from "../../../components/Navigation/BreadCrumb.astro";
+import BreadCrumb, {
```

Comment:
This shouldnt be here, looks like maybe a rebase is needed. This change is from an older PR